<?xml version="1.0"?>

<launch>
<param name="use_sim_time" value="true"/>
<param name="robot_description" command="$(find xacro)/xacro --inorder $(find pluto)/urdf/pluto_good.xacro"/>  
 
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
 
  <!--node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/-->
 
  <!--node name="rviz" pkg="rviz" type="rviz" required="true"/-->
  
  <!--Gazebo empty world launch file-->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    	<arg name="debug" value="false" />
    	<arg name="gui" value="true" />
    	<arg name="paused" value="false"/>
    	<arg name="use_sim_time" value="false"/>
    	<arg name="headless" value="false"/>
    	<arg name="verbose" value="true"/>
  </include>

  <!--Gazebo Simulator-->
  <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model pluto_good.xacro -x 0 -y 0 -z 0.5 " output="screen"/>
  
    <!--node name="robot_controller" pkg="pluto" type="robot_controller"/-->
    <rosparam command="load" file="$(find pluto)/config/joint_state_controller.yaml" />

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/pluto" args="joint_state_controller"/>
  

 
</launch>
